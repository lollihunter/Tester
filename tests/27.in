20 5 20
1 1
9 1
12 1
3 3
5 4
2 5
12 2
18 5
8 1
11 1
5 1
1 4
17 3
15 5
10 2
8 4
1 2
4 3
19 5
20 5
