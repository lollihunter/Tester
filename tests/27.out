10
1 4 11 16 15 7 14 13 19 20 
